include/bluetooth/bluetooth.h

```diff
@@ -310,6 +316,16 @@ struct bt_le_adv_param {
 					    BT_GAP_ADV_FAST_INT_MIN_2, \
 					    BT_GAP_ADV_FAST_INT_MAX_2)
 
+
+#define BT_LE_ADV_CONN_DIR_LOW_DUTY \
+	BT_LE_ADV_PARAM(BT_LE_ADV_OPT_CONNECTABLE | BT_LE_ADV_OPT_ONE_TIME | \
+			BT_LE_ADV_OPT_DIR_MODE_LOW_DUTY, \
+			BT_GAP_ADV_FAST_INT_MIN_2, BT_GAP_ADV_FAST_INT_MAX_2)
+
+#define BT_LE_ADV_CONN_DIR BT_LE_ADV_PARAM(BT_LE_ADV_OPT_CONNECTABLE | \
+					   BT_LE_ADV_OPT_ONE_TIME, 0, 0)
```

Comment:
Should BT_LE_ADV_OPT_ONE_TIME be implied by doing directed advertising? What if someone forgets to set it? I have a feeling that the places that are checking `if (ONE_TIME)` should be changed to `if (ONE_TIME || directed)`.

Reply:
Yes, I can add guard in ```valid_adv_params``` function, so it is not possible to start directed advertising without ```ONE_TIME``` option set

Reply:
I think I'd prefer the implied approach, i.e. bt_le_adv_start_internal() would set BT_DEV_KEEP_ADVERTISING if either ONE_TIME was set or a non-NULL peer_addr was given.

Reply:
Have the options BT_LE_ADV_OPT_CONNECTABLE and BT_LE_ADV_OPT_ONE_TIME implicit, API bt_conn_create_slave_le does not need these to be set.